packages/@lwc/features/src/flags.ts

```diff
@@ -0,0 +1,5 @@
+type FeatureFlag = boolean | null;
```

Comment:
in this file... at the very top, we should make sure that LWC_config exists... and it has a `features` record as well, even if it is just empty, this will guarantee that anyone importing from config is going to get that set up on front, not need to be defensive... the tree-shaking mechanism will preserve initializations.